src/Helpers/babylon.particleHelper.ts

```diff
@@ -0,0 +1,164 @@
+module BABYLON {
+    /**
+     * ParticleSystemType
+     */
+    export enum ParticleSystemType {
+        /**
+         * None is to represents an error in parsing the type string in the create method.
+         */
+        None,
+        /**
+         * Fire particle system.
+         */
+        Fire,
+        /**
+         * Smoke particle system.
+         */
+        Smoke
+    }
+    /**
+     * This class is made for on one-liner static method to help creating particle systems.
+     */
+    export class ParticleHelper {
+        /**
+         * Base Assets URL.
+         */
+        private static _baseAssetsUrl = "https://assets.babylonjs.com/particles/";
+
+        private static _scene: Scene;
+
+        private static _emitter: AbstractMesh;
+
+        /**
+         * This is the main static method (one-liner) of this helper to create different particle systems.
+         * @param type This string will be parsed to a ParticleSystemType
+         * @param emitter The object where the particle system will start to emit from.
+         * @param scene The scene where the particle system should live.
+         * @param gpu If the system will use gpu.
+         * @returns the ParticleSystem created.
+         */
+        public static Create(type: string, emitter: AbstractMesh, scene: Nullable<Scene> = Engine.LastCreatedScene, gpu: boolean = false): ParticleSystem {
```

Comment:
I would rather prefer having a json file loaded where you can find fire or smoke parameters

Does it make sense?

Reply:
Create will be async and you can just call Tools.LoadFile to get the content of the json file (from 
baseAssetsUrl + "/systems.json" for instance)
You can rely on promise to get a CreateAsync function for isntance

Reply:
Oh okay, i haven't see that, thanks you :)

Reply:
But, can I know why you prefer async method + json file over a simple synchronous method?